src/cf-tables/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-tables",
-  "version": "1.1.0",
```

Comment:
@KimberlyMunoz @sebworks Correct, leave package.json at `1.1.0` and update CHANGELOG.md with something like `- **cf-tables:** [MAJOR] Atomized cf-tables.`